src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/Notification.java

```diff
@@ -0,0 +1,131 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * The Notification class represents a complete Notification.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public class Notification implements Runnable {
```

Comment:
i mean storing the record as a private variable and exposing it via getDBInstance or something comparable.